### DISCUSSION TITLE
getServerSideProps cached at Vercel

### DISCUSSION THREAD

**Discussion by dizzyn** [vercel/next.js#60885] [Help] [answered]
### Summary

The `getServerSideProps` seems to be cached unexpectedly.

**Context:** _I have at `Next.js 14.0.4` an admin function "login as user". That works by modifying cookies._ 

**The Issue:**
1. I have a page `/profile` that unwrap cookies and serves the `userId` as a prop into the page.
2. I have changed the cookie by redirect to /api/login and back to the page root `/`
3. The user navigates to the `/profile` page and gets **outdated** version of the page

The `getServerSideProps` have not been called by the navigation:
a. It is called in the dev mode
b. It is called in prod mode on local
c. It is not called on Vercel in production

Please note that I have doisabled browser cache by a HTTP header and the oudated data page JSON is servered from the Vercel and the `getServerSideProps` has not been called.

How i can disable the server side cache if there is any or is that a bug please?

**Comment by icyJoseph** [comment, marked as answer]
I seem to recall a couple of ways in which this can happen.

We should discard `next-pwa`, do you have enabled? Meanwhile I can search through the discussions to find the other case.

**Reply by dizzyn** [reply]
Thank You, i do not have the PWA

**Reply by dizzyn** [reply]
Oh, there is a forgotten service worker, that is it!

**Reply by dizzyn** [reply]
Thank you !
